constants.config.js

```diff
@@ -1727,6 +1727,24 @@ module.exports = {
                 TN : "https://www.ovh.com/tn/hebergement-web/",
                 WE : "http://www.ovh.com/us/"
             },
+            cloudweb_order : {
+                FR : "https://www.ovh.com/fr/order/domain/#/legacy/domain/search?options=~(hosting~'cloudweb1)",
```

Comment:
Didn't you already have the base URL somewhere where you could append `/order/domain/#/legacy/domain/search?options=~(hosting~'cloudweb1)` each time ?

Reply:
That would have been wonderful.

Reply:
You're wonderful baby ❤️

Reply:
:heart: